kolibri/plugins/learn/assets/src/views/ChannelThumbnail.vue

```diff
@@ -19,6 +19,7 @@
 <script>
 
   import ContentIcon from 'kolibri.coreVue.components.ContentIcon';
+  import { ChannelResource } from 'kolibri.resources';
```

Comment:
If the ChannelThumbnail endpoint is working as intended, it should be returning a valid file if fetched - so you should just be able to use the URL directly as a file URL.

If this is not working, we need to make some tweaks, but when it is working correctly, this should require no frontend changes at all.